package.json

```diff
@@ -29,50 +29,39 @@
     "node": ">=4.8.4"
   },
   "scripts": {
-    "test": "grunt test"
+    "clean": "rm -rf package-lock.json && rm -rf node_modules && rm -rf coverage",
+    "test": "mocha --recursive 'test/**/*.js' --reporter spec --timeout 3000 --ui bdd --exit",
+    "test:watch": "npm run test -- -w",
+    "lint": "jshint lib/**/*.js test/**/*.js --config .jshintrc",
+    "test:coverage": "istanbul cover _mocha -- --recursive 'test/**/*.js' --reporter spec --exit",
+    "watch:lib": "watch 'npm run lint && npm test' ./lib"
   },
   "dependencies": {
-    "async": "1.5.2",
-    "body-parser": "~1.11.0",
+    "async": "2.6.1",
+    "body-parser": "~1.18.3",
     "command-shell-lib": "1.0.0",
-    "express": "~4.11.2",
-    "jison": "0.4.17",
-    "logops": "1.0.0",
-    "moment": "~2.19.4",
+    "express": "~4.16.4",
+    "jison": "0.4.18",
+    "logops": "2.1.0",
+    "moment": "~2.22.2",
     "moment-timezone": "~0.5.21",
-    "mongodb": "2.2.35",
-    "mongoose": "4.13.12",
+    "mongodb": "3.1.8",
+    "mongoose": "5.3.6",
     "mu2": "~0.5.20",
-    "mustache": "2.2.1",
+    "mustache": "3.0.0",
     "node-uuid": "~1.4.1",
-    "query-string": "4.3.4",
-    "request": "2.39.0 - 2.81.0",
+    "query-string": "6.2.0",
+    "request": "2.88.0",
     "revalidator": "~0.3.1",
-    "sax": "~0.6.0",
-    "underscore": "~1.7.0",
-    "xmldom": "0.1.19"
+    "underscore": "~1.9.1",
+    "xmldom": "0.1.27"
```

Comment:
"sax" dependency has been removed, I understand that is not currently used by the code. 

What about "xmldom"? Ancient versions of NGSI were based in XML but as far as I understand, none of the APIs used by IOTAs is using XML. Is this still in use?

Whatever the case I think it would be a good idea to add an entry "Remove: old unused dependencies (sax, grunt, xmldom)" or ""Remove: old unused dependencies (sax, grunt)" should be added to CNR.

Reply:
As you can see in the screenshot, it still covered by the current tests:
![dom_coverage](https://user-images.githubusercontent.com/11388886/47677610-e01c7d00-dbbf-11e8-8c81-a3f545385af0.png)
In particular, the unit tests that use XML payloads are:
- https://github.com/telefonicaid/iotagent-node-lib/blob/master/test/unit/lazyAndCommands/lazy-devices-test.js#230
- https://github.com/telefonicaid/iotagent-node-lib/blob/master/test/unit/lazyAndCommands/lazy-devices-test.js#298

I think that we should delete this part of the code in this PR or in a separate one. To the best of my knowledge, XML is not used by any IOTA...What do you think?

Reply:
This PR is a bit "overloaded" so maybe it is better to create and issue and deal with it in a separate PR.